flink-kubernetes-operator-autoscaler/src/main/java/org/apache/flink/kubernetes/operator/autoscaler/JobVertexScaler.java

```diff
@@ -221,7 +221,7 @@ private boolean detectIneffectiveScaleUp(
                 message);
 
         if (conf.get(AutoScalerOptions.SCALING_EFFECTIVENESS_DETECTION_ENABLED)) {
-            LOG.info(
+            LOG.warn(
```

Comment:
Why should this be warn? This looks more like something that should be reflected in the status?

Reply:
There is already an event triggered for this, that should be enough for most users. The log just outputs some additional info